### DISCUSSION TITLE
API?

### DISCUSSION THREAD

**Discussion by nikb72** [MindStudioOfficial/livescopestv#4] [Ideas]
Hi would it be possible to add an API to livescope. This would assist with integration to a bitfocus companion/Birddog camera setup.
Being able to select inputs via their ndi name using the API & change the type of scope/view would be great.

**Comment by MindStudioOfficial** [comment]
Hi,
Yes I agree, an API would be a nice addition to the software. I am currently working on the PRO version of the software and I plan to add an API to this version.

Are you looking for a specific kind of API e.g  HTTP and JSON based or something else?

**Comment by nikb72** [comment]
Json would be fine, would easily fit with the capabilities of bitfocus companion then. Please let me know if you need beta testing when you’re ready. RegardsNik Blasdale07976560968Sent from my iPhone apologies for any brevity or spelling mistakes. On 14 May 2024, at 16:18, Marc Julien Bach ***@***.***> wrote:﻿
Hi,
Yes I agree, an API would be a nice addition to the software. I am currently working on the PRO version of the software and I plan to add an API to this version.
Are you looking for a specific kind of API e.g  HTTP and JSON based or something else?

**Comment by MindStudioOfficial** [comment]
which parts of the software would you like to be able to control through the API?
